created registration page

Nimishaer opened the pull request.

Nimishaer committed:
created registration page

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+
+  const schema = yup.object().shape({
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
+    email: yup.string().required("Email is required*").matches(/^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/g, 'Email must be valid'),
```
Use single quotes to represent string values.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    .min(10, 'Mobile number must be 10 digits')
+    .max(10, 'Mobile number must be 10 digits')
+    .matches(/^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/, 'Enter valid mobile number'),
+    yearOfExperience: yup.string().required("Field Required*"),
```
Use single quotes to represent string values.

Nimishaer replied:
added single quotes to represent string value.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    .max(10, 'Mobile number must be 10 digits')
+    .matches(/^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/, 'Enter valid mobile number'),
+    yearOfExperience: yup.string().required("Field Required*"),
+    education: yup.string().required("Field required*"),
```
Use single quotes to represent string values.

Nimishaer replied:
added single quotes to represent string value.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    .matches(/^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/, 'Enter valid mobile number'),
+    yearOfExperience: yup.string().required("Field Required*"),
+    education: yup.string().required("Field required*"),
+    dob: yup.string("Enter valid date of birth").required("Field required*"),
```
Use single quotes to represent string values.

Nimishaer replied:
added single quotes to represent string value.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    yearOfExperience: yup.string().required("Field Required*"),
+    education: yup.string().required("Field required*"),
+    dob: yup.string("Enter valid date of birth").required("Field required*"),
+    areaOfExpertise: yup.string().required("Field required*"),
```
Use single quotes to represent string values.

Nimishaer replied:
added single quotes to represent string value.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    education: yup.string().required("Field required*"),
+    dob: yup.string("Enter valid date of birth").required("Field required*"),
+    areaOfExpertise: yup.string().required("Field required*"),
+    password: yup.string().required("Password cannot be empty*").matches(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/, 'Minimum eight characters, at least one letter and one number'),
```
Use single quotes to represent string values.

Nimishaer replied:
added single quotes to represent string value.

mishelshaji commented on client/src/pages/instructor/Registration.js:
```diff
@@ -0,0 +1,103 @@
+    dob: yup.string("Enter valid date of birth").required("Field required*"),
+    areaOfExpertise: yup.string().required("Field required*"),
+    password: yup.string().required("Password cannot be empty*").matches(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/, 'Minimum eight characters, at least one letter and one number'),
+    confirmPassword: yup.string().oneOf([yup.ref('password')], "Passwords doesn't match").required("Confirm password cannot be empty*")
```
Use single quotes to represent string values.

Nimishaer replied:
Added single quotes to represent string value.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+
+  const schema = yup.object().shape({
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
+    email: yup.string().required("Email is required*").matches(/^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/g, 'Email must be valid'),
```
Use single quotes to represent string values.

Nimishaer replied:
Added single quotes to represent string value.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+  }
+
+  const schema = yup.object().shape({
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
```
Use single quotes to represent string values.

Nimishaer replied:
Added single quotes to represent string value.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+  const schema = yup.object().shape({
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
+    email: yup.string().required("Email is required*").matches(/^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/g, 'Email must be valid'),
+    mobile: yup.string().required("Mobile is required*").min(10,'Mobile number must be 10 digits').max(10,'Mobile number must be 10 digits').matches(/^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/,'Enter valid mobile number'),
```
Use single quotes to represent string values.

Nimishaer replied:
Added single quotes to represent string value.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+    name: yup.string().matches(/^[A-Za-z ]*$/, 'Please enter valid name').required("Name field cannot be empty*"),
+    email: yup.string().required("Email is required*").matches(/^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/g, 'Email must be valid'),
+    mobile: yup.string().required("Mobile is required*").min(10,'Mobile number must be 10 digits').max(10,'Mobile number must be 10 digits').matches(/^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/,'Enter valid mobile number'),
+    password: yup.string().required("Password cannot be empty*").matches(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/,'Minimum eight characters, at least one letter and one number'),
```
Use single quotes to represent string values.

Nimishaer replied:
Added single quotes to represent string value.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+                {errors.name ? <span className="text-danger">{errors.name.message}</span> : null}
+              </div>
+              <div className="mb-3">
+                <label htmlFor="">Email</label>
```
Give a value for the htmlFor property.

Nimishaer replied:
Given value for htmlFor property.

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+                {errors.email ? <span className="text-danger">{errors.email.message}</span> : null}
+              </div>
+              <div className="mb-3">
+                <label htmlFor="">Mobile</label>
```
Give a value for the htmlFor property.

Nimishaer replied:
Given value for htmlFor property

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+                {errors.mobile ? <span className="text-danger">{errors.mobile.message}</span> : null}
+              </div>
+              <div className="mb-3">
+                <label htmlFor="">Password</label>
```
Give a value for the htmlFor property.

Nimishaer replied:
Given value for htmlFor property

mishelshaji commented on client/src/pages/student/Registration.js:
```diff
@@ -0,0 +1,70 @@
+                {errors.password ? <span className="text-danger">{errors.password.message}</span> : null}
+              </div>
+              <div className="mb-3">
+                <label htmlFor="">Confirm Password</label>
```
Give a value for the htmlFor property.

Nimishaer replied:
Given value for htmlFor property

Nimishaer committed:
Fixed errors

Used single quotes to represent string values.
Provided values for htmlFor property
